editor.planx.uk/src/@planx/components/MapAndLabel/model.ts

```diff
@@ -0,0 +1,16 @@
+import { MoreInformation, parseMoreInformation } from "../shared";
+
+export interface MapAndLabel extends MoreInformation {
+  fn: string;
+  title?:string;
+  description?:string;
+}
+
+export const parseContent = (
+  data: Record<string, any> | undefined,
+): MapAndLabel => ({
+  fn: data?.fn || "",
+  ...parseMoreInformation(data),
+  title:"",
+  description:""
+});
```

Comment:
Similar to above: 
- title & description should reference their respective form inputs - eg `title: data?.title || ""` & `description: data?.description` (no fallback necessary if optional field)
- order according to editor form for readability, and keep any spread (`...`) values last !